### PR TITLE
Extend AppNavigationCaption with slot for custom actions icon

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
Required for https://github.com/nextcloud/calendar/issues/3965

### CODE REVIEW

**Review by marcoambrosini — COMMENTED** [review]
It doesn't seem to work in the docs, or maybe I'm missing something?

There shouldn't be the need to modify the component to pass in a mdi icon into the slot. it should work with

```
<template #actions>
    <Plus #icon /> // This icon will be in the trigger of the action menu instead of the 3 dots
    <ActionButton />
    .... all other action elements
<template>
```

See https://deploy-preview-2587--nextcloud-vue-components.netlify.app/#/Components/Actions

**Comment by JuliaKirschenheuter** [comment]
It should be right now. Please review ;)

File: src/components/AppNavigationCaption/AppNavigationCaption.vue [hunk 1]
```diff
@@ -7,6 +7,23 @@
 	</template>
 </AppNavigationCaption>
 ```
+
+### Element with a slot for custom actions icon
+```
+<AppNavigationCaption
+	title="Your caption goes here">
+	<template #actionsTriggerIcon>
+		<PlusIcon slot="icon" :size="20" />
```
Comment (raimund-schluessler) [inline src/components/AppNavigationCaption/AppNavigationCaption.vue:16]: The `PlusIcon` component is missing and not showing. You need to import it. Have a look at e.g. https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/Actions/Actions.vue#L88-L122 for how to.

Comment (JuliaKirschenheuter) [reply src/components/AppNavigationCaption/AppNavigationCaption.vue:16]: ok, something went wrong =(
actually i thought, that it is already imported.
Thank you!

**Comment by raimund-schluessler** [comment]
Works as it should now.

But I have to admit, I am a bit puzzled why the proposal of @marcoambrosini does not (I checked it in the docs now, and the icon does not render):
> ```
> <template #actions>
>     <Plus #icon /> // This icon will be in the trigger of the action menu instead of the 3 dots
>     <ActionButton />
>     .... all other action elements
> <template>
> ```

**Comment by JuliaKirschenheuter** [comment]
> Works as it should now.
> 
> But I have to admit, I am a bit puzzled why the proposal of @marcoambrosini does not (I checked it in the docs now, and the icon does not render):
> 
> > ```
> > <template #actions>
> >     <Plus #icon /> // This icon will be in the trigger of the action menu instead of the 3 dots
> >     <ActionButton />
> >     .... all other action elements
> > <template>
> > ```

Because i need to put somehow custom icon slot to Actions via ```<template #icon>```.  There were no possibility at the AppNavigationCaption before.

**Pull request merged**